src/main/java/com/serenitydojo/calculator/Calculator.java

```diff
@@ -1,7 +1,90 @@
 package com.serenitydojo.calculator;
 
+import static org.apache.commons.lang3.StringUtils.isNumeric;
+
 public class Calculator {
+    private String operator = "";
+    private int evaluation = 0;
+
     public int evaluate(String expression) {
+        if (!expression.isEmpty()) {
+            return iteratingThrewTheExpression(expression);
+        }
+
         return 0;
     }
+
+    private void ifNotAnOperatorCallIllegalMathsOperatorException() {
+
+
+
+    }
+
+    private int doingTheOperation( int number1, int number2) throws IllegalMathsOperatorException{
+        System.out.println("---" + this.operator + "---");
+        ifNotAnOperatorCallIllegalMathsOperatorException();
+
+        if(this.operator.matches("(-|/|\\+|\\*)") || this.operator == "") {
+            switch (this.operator) {
+                case "+":
+                    return number1 + number2;
+                case "-":
+                    return number1 - number2;
+                case "/":
+                    return number1 / number2;
+                case "*":
+                    return number1 * number2;
+                default:
+                    return number1;
+            }
+        }
+        else {
+            throw new IllegalMathsOperatorException("Unknown operator " + this.operator);
+        }
+    }
+
+    private int iteratingThrewTheExpression(String expression) {
+        String[] elements = expression.split(" ");
+            for (String element : elements) {
+                preparingTheCalculation(element);
+            }
+
+            return this.evaluation;
+    }
+
+    private void preparingTheCalculation(String element) {
+        if (isNumeric(element)) {
+            int[] intFromString = gettingIntFromString(element);
+            this.evaluation = doingTheOperation(intFromString[0], intFromString[1]);
+        } else {
+            this.operator = element;
+        }
+    }
+
+
+    private int[] gettingIntFromString(String element) {
```

Comment:
This method is a bit confusing. Methods tend to be easier to understand if they have only use input parameters, and avoid using fields, as it's hard to know where the field values (e.g. `evaluation` in this case) are doing.

Reply:
Ok, I understand. Will try to avoid field in the future. Thx.